src/Assets/Items/Food/Apple/apple.xob.meta

```diff
@@ -1,5 +1,5 @@
 MetaFileClass {
- Name "{D0566D3CAE967F4D}Assets/Items/Food/Apple/apple.xob"
+ Name "{D0566D3CAE967F4D}Assets/Items/Food/Apple/Appleasdasd.xob"
```

Comment:
Same name issue as commented above. Not sure how exactly this happened. You could try to manually edit the path in the meta file I guess